01-node-tutorial/answers/03-modules.js

```diff
@@ -0,0 +1,11 @@
+const names = require('./04-names');
+const sayHello = require('./05-utils');
+const personData = require('./06-alternative-flavor')
+
+console.log(names);
+console.log(sayHello);
+console.log(personData);
+
+sayHello(names.name1)
+sayHello(names.name2)
+require('./07-mind-grenade')
```

Comment:
Just as a heads up this red circle just means github wants an empty line at the end of every file.